README.md

```diff
@@ -1,72 +1,92 @@
-# Devboxes
+# CES-Devboxes
+
+This repository contains ansible roles to setup a new development machine for the Cloudogu EcoSystem developers.
 
-This repository contains ansible roles to setup my development machines and helps to keep them in sync.
-The ansible playbook will install software which i need for my day to day work as developer.
-It will also install my set of configuration for those tools.
 The playbook will install and configure the following tools:
 
-* zsh (configured with [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh))
-* tmux (inspired from [.tmux](https://github.com/gpakosz/.tmux))
-* vim (configured with [vimrc](https://github.com/amix/vimrc))
-* docker
-* vagrant
-* virtual box
-* packer
-* terraform
-* gcloud, kubectl and kubectx
-* gpg
-* git
-* mercurial
-* java and maven
-* node.js
-* golang
-* vscode
-* Intellij
-* and many more
-
-## Supported Operating Systems
-
-* Ubuntu (tested on 19.04)
-* Manjaro Linux
-* Mac OS X
+**IDEs**
 
-## Prerequisites
+- IntelliJ Ultimate
+- VSCode with some convenient plugins
 
-* Ansible >= 2.6
+**Working with VMs**
 
-### MacOSX
+- Virtualbox (supports Secure-Boot)
+- VMWare Workstation Pro 16 (supports Secure-Boot)
+- Hashicorp
+  - packer
+  - terraform
+  - vagrant
 
-* Install [homebrew](https://brew.sh/)
-* Install ansible `brew install ansible`
+**Terminal**
 
-### Ubuntu
+Installs zsh with some preconfigured things and plugins. See [.zshrc](config/zshrc) for more information.
 
-Install ansible
+**Browser**
 
-```bash
-sudo add-apt-repository ppa:ansible/ansible
-sudo apt-get update
-sudo apt-get install ansible
-```
+The devbox install Chrome and Firefox as Browsers. It also adds some basic extensions and bookmarks into Chrome.
 
-### Manjaro
+**SDK Manager**
 
-Install ansible `sudo pacman -S ansible`
+- Jabba with preinstalled Java8 and Java11
+  - Also adds two aliases `java8` and `java11` for a convenient switch between versions.
+- gvm with newest Go version being installed
+- nvm with newest Node version being installed
 
-## Getting started
+**k8s**
+
+- kubectl
+- gcloud
+- k9s
+
+**Recording tools**
+
+- gif image optimization: `gifsicle`
+- gif screen recording: `peek`
+- video screen recording: `kazam`
 
-Just run the `devbox` script, on the first run it will ask some question to personalize the installed configurations.
-After that it will ask for your sudo password and then apply the ansible playbook. 
+**Snippet Manager with a preset of handy snippets**
 
-## Tags
+The snippet manager pet is installed. You can edit your snippets with `pet edit` and search and apply snippets in the
+CLI with the hotkey `CTRL+S`.
 
-Its also possible to only apply certain tags, e.g.
+## Tested Operating Systems
 
-`devbox --tags zsh`
+* Ubuntu (tested on 22.04)
 
-See [playbook](playbook.yml) for existing tags.
+## Prerequisites
+
```

Comment:
You also need a gpg key installed if you want to use its key ID in the vars.yml

Reply:
Done